cmake/process_users_input.cmake

```diff
@@ -218,3 +219,11 @@ endfunction()
 
 cast_log_level_to_int(LOG_LEVEL LOG_LEVEL)
 cast_log_level_to_int(LOG_LEVEL_MASTER LOG_LEVEL_MASTER)
+
+if (PROXY_PYBINDING)
+    set(EXTRA_FLAGS -fPIC)
```

Comment:
Can you rename EXTRA_FLAGS to something like EXTRA_CXX_FLAGS or so? Was pretty confused by the name.

Reply:
done